backend/service/materials.service.ts

```diff
@@ -0,0 +1,13 @@
+import express, {Request, Response } from "express";
+import  {Material} from "../models/material.model";
+
+export const getAllMaterials = async () => {
+  return Material.find()
+}
+
+const createMaterial = async (input: any) => {
```

Comment:
The input should not be of type `any` here as you know the shape of the input data because you defined the material schema - I also gave you that youtube video that tells you how to create a typed schema using `zod`